<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ვალუტის კონვერტაცია</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>

  <!-- Title Label -->
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>600</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 26pt; font-weight: bold; color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>Currency Converter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>

  <!-- Amount Label -->
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>

  <!-- Amount LineEdit -->
  <widget class="QLineEdit" name="amountLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter amount</string>
   </property>
  </widget>

  <!-- From Currency ComboBox -->
  <widget class="QComboBox" name="fromCurrencyComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14pt; padding: 5px; border: 1px solid rgb(150, 150, 150); border-radius: 5px;</string>
   </property>
   <item>
    <property name="text">
     <string>EUR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GEL</string>
    </property>
   </item>
  </widget>

  <!-- To Currency ComboBox -->
  <widget class="QComboBox" name="toCurrencyComboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14pt; padding: 5px; border: 1px solid rgb(150, 150, 150); border-radius: 5px;</string>
   </property>
   <item>
    <property name="text">
     <string>EUR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GEL</string>
    </property>
   </item>
  </widget>

  <!-- Convert Button -->
  <widget class="QPushButton" name="convertButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 150, 255); color: white; font-size: 16pt; border-radius: 10px; padding: 10px;</string>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>

  <!-- Result Label -->
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Result will be displayed here</string>
   </property>
  </widget>

  <!-- Logout Button (Upper) -->
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 100, 100); color: white; font-size: 12pt; border-radius: 5px; padding: 5px;</string>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
